### PR TITLE
Deploy Mumbai

### PR DESCRIPTION
**Pull request by scott-silver**
It turns out that when a Github Action adds a commit to a branch, it doesn't trigger the same CI actions as when you push changes. Long thread about it [here](https://github.com/orgs/community/discussions/25702).

So I had to add a vacuous change to trigger CI checks after the new roots were added.

That seems like an important issue, because it means scenarios aren't run against newly-deployed roots.

### CODE REVIEW

**Comment by jflatow** [comment]
> It turns out that when a Github Action adds a commit to a branch, it doesn't trigger the same CI actions as when you push changes. Long thread about it [here](https://github.com/orgs/community/discussions/25702).
> 
> So I had to add a vacuous change to trigger CI checks after the new roots were added.
> 
> That seems like an important issue, because it means scenarios aren't run against newly-deployed roots.

I had actually thought of this as almost a feature since we had already run deploy in scenarios and the workflow must/should have been run on a branch passing CI, but I agree it could be good to run again with real roots before merge

File: README.md
Comment (kevincheng96) [inline README.md:256]: nit: should we revert this change?

**Review by kevincheng96 — APPROVED** [review]
Awesome, can't wait to try it out!

File: README.md
Comment (scott-silver) [reply README.md]: sure thing. reverted

**Comment by scott-silver** [comment]
We can merge this set of roots, but we have several pending patches to Comet that will not be included (#596 and the incoming sweep functionality). I have an in-progress change that will make some small changes to the Bulker as well.

We can either merge these roots and then run a migration to upgrade once those patches are merged. Or we can just wait until those patches are merged and then redeploy.

**Comment by jflatow** [comment]
> We can merge this set of roots, but we have several pending patches to Comet that will not be included (#596 and the incoming sweep functionality). I have an in-progress change that will make some small changes to the Bulker as well.
> 
> We can either merge these roots and then run a migration to upgrade once those patches are merged. Or we can just wait until those patches are merged and then redeploy.

Let's wait until all the patches are all ready, one of the goals is to have the same version of Comet everywhere, and its nice to avoid the unnecessary migration

**Comment by scott-silver** [comment]
deleted the old roots, rebased and added new roots: https://github.com/compound-finance/comet/actions/runs/3373881750/jobs/5598995780

**Pull request merged**
